File: ed2/gara1/olimpo/att/olimpo.c
#include <stdio.h>
#include <assert.h>

#define MAXN 1000000

int falsifica(int N, char Q[], int A[]) {
    // Mettete qui il codice della soluzione
    return 42;
}


char Q[MAXN];
int  A[MAXN];

int main() {
    FILE *fr, *fw;
    int N, i;

    fr = fopen("input.txt", "r");
    fw = fopen("output.txt", "w");
    assert(1 == fscanf(fr, "%d", &N));
    Q[0] = 'X';
    A[0] = -1;
    for(i=1; i<N; i++) {
        assert(1 == fscanf(fr, " %c", &Q[i]));
        assert(1 == fscanf(fr, "%d", &A[i]));
    }

    fprintf(fw, "%d\n", falsifica(N, Q, A));
    fclose(fr);
    fclose(fw);
    return 0;
}

File: ed2/gara1/olimpo/att/olimpo.pas
function falsifica(N: longint; var Q: array of char; var A: array of longint): longint;
begin
    (* Mettete qui il codice della soluzione *)
    falsifica := 42;
end;


const
    MAXN = 1000000;
var
    N, i  : longint;
    Q     : array[0..MAXN-1] of char;
    A     : array[0..MAXN-1] of longint;
    fr, fw: text;

begin
    assign(fr, 'input.txt');
    assign(fw, 'output.txt');
    reset(fr);
    rewrite(fw);
    readln(fr, N);
    Q[0] := 'X';
    A[0] := -1;
    for i:=1 to N-1 do
        readln(fr, Q[i], A[i]);

    writeln(fw, falsifica(N, Q, A));
    close(fr);
    close(fw);
end.
